saleor/static/dashboard-next/components/VisibilityCard/VisibilityCard.tsx

```diff
@@ -0,0 +1,111 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import {
+  createStyles,
+  Theme,
+  withStyles,
+  WithStyles
+} from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import * as React from "react";
+
+import CardTitle from "../../components/CardTitle";
+import ControlledSwitch from "../../components/ControlledSwitch";
+import { FormSpacer } from "../../components/FormSpacer";
+import i18n from "../../i18n";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    date: {
+      "& svg": {
+        fill: theme.palette.primary.main
+      },
+      marginTop: theme.spacing.unit * 4
+    },
+    expandedSwitchContainer: {
+      marginBottom: 0
+    },
+    switchContainer: {
+      marginBottom: -theme.spacing.unit
+    }
+  });
+
+interface VisibilityCardProps extends WithStyles<typeof styles> {
+  children?: React.ReactNode;
```

Comment:
`React.ReactNode | React.ReactNodeArray` would be better since this component passes no props to its children